Classes/ReportsConfigurationStatus.php

```diff
@@ -58,9 +58,9 @@ public function getLangModes() {
 			'fail' => array(),
 		);
 		$value = $GLOBALS['LANG']->sL('LLL:EXT:languagevisibility/locallang_db.xml:reports.ok.value');
-		$severity = tx_reports_reports_status_Status::OK;
+		$severity = \TYPO3\CMS\Reports\Status::OK;
 
-		$rootTpls = \TYPO3\CMS\Backend\Utility\BackendUtility::getRecordsByField('sys_template', 'root', '1', '', 'pid');
```

Comment:
.. but generally when I read the code I understand it that it should be desired to check all templates, not only one per parent